src/main/java/com/codurance/retropolis/web/config/WebConfig.java

```diff
@@ -3,6 +3,7 @@
 import static java.util.Collections.singletonList;
```

Comment:
i don't think we need this import anymore

Reply:
Cool! Thanks Ewan!